src/utilities/HYPRE_utilities.h

```diff
@@ -174,7 +164,24 @@ typedef double       hypre_double;
 typedef float        hypre_float;
 typedef long double  hypre_long_double;
 
-/**@}*/
+/*--------------------------------------------------------------------------
+ * Sequential MPI stuff
+ *--------------------------------------------------------------------------*/
+
+#ifdef HYPRE_SEQUENTIAL
+typedef HYPRE_Int MPI_Comm;
+#endif
+
+/*--------------------------------------------------------------------------
+ * HYPRE AP user functions
+ *--------------------------------------------------------------------------*/
+
+/* Checks whether the AP is on */
+/* TODO (VPM): this function is provided for backwards compatibility
+   and will be removed in a future release */
+HYPRE_Int HYPRE_AssumedPartitionCheck(void);
```

Comment:
I'm wondering if it's a good time to remove this function...

Reply:
Probably.  I didn't even know it existed or why users would need it.  Seems more like a developer utility to me.  Do you know anything more about it?

Reply:
I think it dates back to the global partition option that we removed a few years ago. It was a runtime check for that option, now it just returns true because we don't have the global partition option anymore. MFEM's code has several mentions to it (e.g., [here](https://github.com/mfem/mfem/blob/cd2715680ffff566e0be5d856da229bb0a27b91a/linalg/hypre.hpp#L245-L251)), if we remove it we can probably make things a bit cleaner for them. cc @v-dobrev @tzanio

Reply:
FYI, I've made this change.